pkgs/development/python-modules/fyta-cli/default.nix

```diff
@@ -18,12 +19,15 @@ buildPythonPackage rec {
     owner = "dontinelli";
     repo = "fyta_cli";
     rev = "refs/tags/v${version}";
-    hash = "sha256-V6yf5XFPPePQkRKyH6pyVFDsviYjnIs9g+PavCTDiZo=";
+    hash = "sha256-U4UvnHK5osMUYvfIRL2Tew+2aJrs9xelK46igPw5l7E=";
   };
 
   build-system = [ hatchling ];
 
-  dependencies = [ aiohttp ];
+  dependencies = [
+    aiohttp
+    mashumaro
+  ];
 
   # Module has no tests
   doCheck = false;
```

Comment:
There are now tests.

Reply:
Looks like that I'm still missing a piece to get the tests running.